pyproject.toml

```diff
@@ -17,6 +17,9 @@ dependencies = [
     'jwcrypto~=1.3.1',
     'psutil~=5.8',
     'setproctitle~=1.2',
+
+    'httpx~=0.24.1',
+    'respx~=0.20.1',
```

Comment:
`respx` unused at the moment, but figured I'd leave this for now. Fine to kill it if the noise is distracting.

Reply:
Kill it please. Or we **will** forget and we try to keep the number of dependencies (test or runtime) to the absolute minimum.